ansible_base/authentication/views/ui_auth.py

```diff
@@ -36,22 +36,16 @@ def generate_ui_auth_data():
 
     for authenticator in authenticators:
         if authenticator.category == 'password':
-            response['show_login_form'] = True
             response['passwords'].append(
                 {
                     'name': authenticator.name,
-                    'type': authenticator.type,
                 }
             )
+            response["show_login_form"] = True
         elif authenticator.category == 'sso':
             try:
-                response['ssos'].append(
-                    {
-                        'name': authenticator.name,
-                        'login_url': authenticator.get_login_url(),
-                        'type': authenticator.type,
-                    }
-                )
+                response['ssos'].append({'name': authenticator.name, 'login_url': authenticator.get_login_url(), 'type': authenticator.type.split('.')[-1]})
+                response["show_login_form"] = True
```

Comment:
These seem like more erroneous changes.

Reply:
show login form change is intentional, black flattened the dict.

Reply:
type is needed because the last component chooses the icon.